src/cow-react/common/helpers/pendingOrderSummary.ts

```diff
@@ -0,0 +1,19 @@
+import { Order, OrderKind } from 'state/orders/actions'
+import { formatSmartAmount } from 'utils/format'
+import { CurrencyAmount } from '@uniswap/sdk-core'
+
+export function pendingOrderSummary(order: Order): string {
+  const { kind, buyAmount, sellAmount, inputToken, outputToken, feeAmount } = order
+
+  const inputPrefix = kind === OrderKind.BUY ? 'at most ' : ''
+  const outputPrefix = kind === OrderKind.SELL ? 'at least ' : ''
+
+  const inputAmount = CurrencyAmount.fromRawAmount(inputToken, sellAmount.toString()).add(
+    CurrencyAmount.fromRawAmount(inputToken, feeAmount.toString())
+  )
+  const outputAmount = CurrencyAmount.fromRawAmount(outputToken, buyAmount.toString())
+
+  return `Swap ${inputPrefix}${formatSmartAmount(inputAmount)} ${
```

Comment:
Shouldn't we have a different summary for limit orders?

Reply:
Good question, idk. @elena-zh any comments?

Reply:
It's a good question because we use the word `Swap`, however, I would think is kind of ok. 

If we want to make a change, we could consider using `Sell` and `Buy` instead of `Swap`